src/api/spec/support/capybara.rb

```diff
@@ -1,18 +1,45 @@
-require 'capybara/poltergeist'
 
 Capybara.default_max_wait_time = 6
 Capybara.save_path = Rails.root.join('tmp', 'capybara')
-Capybara.server = :webrick
 
-Capybara.register_driver :poltergeist do |app|
-  Capybara::Poltergeist::Driver.new(app, debug: false, timeout: 30)
+Capybara.configure do |config|
+  config.run_server = false
+  config.app_host = 'http://testapp:3005'
 end
 
-Capybara.register_driver :rack_test do |app|
-  Capybara::RackTest::Driver.new(app, headers: { 'HTTP_ACCEPT' => 'text/html' })
+caps = Selenium::WebDriver::Remote::Capabilities.chrome(
```

Comment:
Why do we need all of this?

Reply:
the caps? copy&paste from blogs - no-sandbox is important for docker, all others might be default anyway - didn't have time to reduce them, at some point the penalty shootout kicked in

Reply:
I'm sure somewhen the games will be over...